cli/src/maintenance.rs

```diff
@@ -88,6 +88,18 @@ pub enum MaintenanceOutput {
         from_stake_seed: u64,
         to_stake_seed: u64,
     },
+
+    UnstakeFromInactive {
```

Comment:
On the one hand there is only one thing that could be inactive here, but still, I would write `UnstakeFromInactiveValidator` to make it more descriptive.